.cubic/wiki/04-section-data/02-p-asset-management.md

```
@@ -54,6 +54,13 @@ When a ZIP archive is uploaded as a project source, the `PostgresProjectStore` p
 
 Sources: [apps/backend/src/projects/postgresProjectStore.ts:1140-1200](../../../apps/backend/src/projects/postgresProjectStore.ts#L1140-L1200), [apps/backend/src/projects/postgresProjectStore.ts:1210-1250](../../../apps/backend/src/projects/postgresProjectStore.ts#L1210-L1250)
 
+### Legacy Snapshot Repair
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)